util/Migrator/DbMigrator.cs

```diff
@@ -121,8 +125,17 @@ private bool MigrateDatabase(bool enableLogging = true,
             .SqlDatabase(_connectionString)
             .WithScriptsAndCodeEmbeddedInAssembly(Assembly.GetExecutingAssembly(),
                 s => s.Contains($".{folderName}.") && !s.Contains(".Archive."))
-            .WithTransaction()
-            .WithExecutionTimeout(new TimeSpan(0, 5, 0));
+            .WithExecutionTimeout(TimeSpan.FromMinutes(5));
+
+        if (_noTransactionMigration)
+        {
+            builder = builder.WithTransactionPerScript()
+                .WithExecutionTimeout(new TimeSpan(0, 60, 0));
```

Comment:
🎨 You can use `TimeSpan.FromMinutes(60)` here like the above.

Reply:
updated